test/markup/index.js

```diff
@@ -26,7 +26,7 @@ function testLanguage(language) {
 
           actual.trim().should.equal(expected.trim());
           done();
-        });
+        }).catch(function(err) { return done(err) });
```

Comment:
Not sure if this was intended to be left there.

Reply:
It's already merged in fact.

Reply:
Yes, this is intentional.  It allows the test runner to properly colorize errors for debugging failing tests.